services/controllerhost/api_handlers.go

```diff
@@ -143,11 +150,20 @@ func areExtentStoresHealthy(context *Context, extent *m.DestinationExtent) bool
 	}
 
 	for _, h := range storeIDs {
+
+		status := "up"
```

Comment:
there is already a enum within dfdd for this. the right thing to do is to add a String() method to that enum. Let me refactor the code a bit to make it cleaner.